srcpkgs/gama/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'gama'
+pkgname=gama
+version=1.0.4
```

Comment:
Hi, I've released new version. And this version includes new description about license. 📄 
```suggestion
version=1.0.5
```